test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -529,15 +529,44 @@ export default class TabularView extends Component {
             consolidatedDataObject.benchmarkNVM = testDataObject.benchmarkNVM;
 
             let higherBetter;
-            try {
-                if (benchmarkVariantsInfo[benchmark][variant][metric]["higherbetter"] === "t") {
-                    higherBetter = true;
-                } else {
-                    higherBetter = false;
-                }
-            } catch (higherBetterNotFoundError) {
+            
+            //get BenchmarkRouter & Metric files from server
+        var benchmarkMetricRequest = async () => {
+            return await fetch( `/api/getBenchmarkVariants`, {
+            method: 'get'
+        } );
+        };
+        var benchmarkMetricVariants = async () => {
+            return await benchmarkMetricRequest.json();
+        };
+        var BenchmarkRouter = benchmarkMetricVariants[0];
+        var BenchmarkMetric = benchmarkMetricVariants[1]; 
```

Comment:
`var BenchmarkRouter = benchmarkMetricVariants['benchmarkMetricRouter'];` might be better.